scripts/ban.sh

```diff
@@ -28,7 +28,7 @@ FORBIDDEN=(
   "expect(false)"
   "expect(true)"
   "from 'lodash'"
-  'from "lodash'
+  'from "lodash"'
```

Comment:
I made a mistake in https://github.com/sebranly/WordyWorld/pull/17
`import get from "lodash/get"` is preferred over `import { get } from "lodash"` for bundle-size reason